pages/ideaDetails/[id].js

```diff
@@ -89,6 +94,9 @@ export default function IdeaDetails({ ideas }) {
         </Hashtags>
         <Link href="/">Go Back</Link>
         <Link href={`/edit/${ideaDetails.id}`}>Edit</Link>
+        <StyledButton onClick={() => onDelete({ id })}>
```

Comment:
```suggestion
        <StyledButton onClick={() => onDelete(id)}>
```
You don't need the curly brackets here or above. 🤔